druntime/src/core/internal/container/hashtab.d

```diff
@@ -218,8 +218,7 @@ private:
 
     void ensureNotInOpApply()
     {
-        if (_inOpApply)
-            assert(0, "Invalid HashTab manipulation during opApply iteration.");
+        assert(!_inOpApply, "Invalid HashTab manipulation during opApply iteration.");
```

Comment:
... but these other ones are not, they are a behavioural change. and will behave differently in release mode.